package.json

```diff
@@ -52,7 +53,7 @@
     "@types/node": "^12.12.42",
     "@types/sinon": "^10.0.6",
     "@types/validator": "^13.1.4",
-    "@typescript-eslint/eslint-plugin": "^5.3.0",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
```

Comment:
Maybe leave this for another PR as well? Just to keep this PR to the point

Reply:
Not sure if I made this change or it was part of a merge from some other pull request but I have reverted to 5.3.0 for now as per your suggestion.